recipes/geoplot/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
+conda upgrade -y -q gdal
```

Comment:
conda command while building are a no no. Also, if the build is only `python setup.py install --single-version-externally-managed --record=record.txt` you can add that to the `meta.yaml` file. (See the recipe example in this repo.)